src/bvals/bvals_var.cpp

```diff
@@ -97,6 +108,7 @@ void BoundaryVariable::InitBoundaryData(BoundaryData<> &bd, BoundaryQuantity typ
     bd.recv[n] =
         BufArray1D<Real>(bd.buffers, std::make_pair(offsets.at(n) + total_size,
                                                     offsets.at(n + 1) + total_size));
+    bd.recv_h[n] = Kokkos::create_mirror_view(bd.recv[n]);
```

Comment:
There are `Kokkos` calls all through the infrastructure code. While it might be useful to hide these behind abstractions, that would be beyond the scope of this PR.